ansible_collections/arista/avd/molecule/deploy_to_cv/intended/structured_configs/base_configs/ci-s1-leaf1.yml

```diff
@@ -0,0 +1,12 @@
+hostname: devel-s1-leaf1
```

Comment:
Not sure if these were created manually but consider renaming to match filename for the host